operator/pkg/apis/core/v1beta1/configconnectorcontext_types.go

```diff
@@ -42,6 +42,18 @@ type ConfigConnectorContextSpec struct {
 	// Specifies the project to use for preconditions, quota and billing.
 	// Should only be used when requestProjectPolicy is set to BILLING_PROJECT.
 	BillingProject string `json:"billingProject,omitempty"`
+
+	// StateIntoSpec is the user override of the default value for the
+	// cnrm.cloud.google.com/state-into-spec annotation if the annotation is
+	// unset in a resource.
+	// 'absent' means unspecified fields in the resource spec stay unspecified
+	// after the successful reconciliations.
+	// 'merge' means the unspecified fields in the resource spec are populated
+	// after the successful reconciliations if the fields are computed/defaulted
+	// by the API.
+	//+kubebuilder:validation:Enum=absent;merge
+	//+kubebuilder:validation:Optional
+	StateIntoSpec *string `json:"stateIntoSpec,omitempty"`
```

Comment:
This looks right, it's important that it's optional so :+1: .  Name ("stateIntoSpec") seems logical and fine, not sure if anyone else on the team has a better suggestion but barring that no objections to the name!

Reply:
Would it be better to call this `StateIntoSpecOverride` ? I know it stutters a bit but I still find the `*IntoSpec` confusing as a newish comer here.

Reply:
I'm a bit hesitant to call it `StateIntoSpecOverride` because it sounds like this value will override the resource-level `state-into-spec` configuration.

The override hierarchy is:
resource-level `state-into-spec` annotation > 
CCC/namespace-level `stateIntoSpec` configuration > 
CCC/cluster-level `stateIntoSpec` configuration.

I don't know if I have any other good ideas so I'm calling for some more feedback @jingyih @gemmahou !

Reply:
I think we probably don't want yet another level of `state-into-spec` per resource Kind. /cc @justinsb